### ISSUE TITLE
Bug: Portrait video cropping on full screen carousel view looks bad

### ISSUE THREAD

**Issue by danieldaquino** [damus-io/damus#2165]
**What happens**

On the full screen carousel view, portrait videos look small, because they are set to "fit" the video player, not "fill"

**What I expect to happen**

Portrait videos should be cropped differently on that view. Preferably the sides of the video should touch the sides of the screen.

**Link to noteID, npub**
Example note: [note1f2uwa5ep9c79er4p7nz993jeaa6zmn6sdjeuehay90q03dj0vnqsztplhr](https://damus.io/nevent1qqsy4w8w6vsju0zu36slf3zjcev77apdeagxev7vm7jzhs8cke8kfsgprdmhxue69uhkummnw3ez6ur4vgh8xetdd9ek7mpwv3jhvqguwaehxw309ahx7um5wghxy6t5vdhkjmn9wgh8xmmrd9skcqg6waehxw309ahx7um5wghx7unpdenk2urfd3kzuer9wcqjuamnwvaz7tmcdckj6ffjxpmhxueddphrvep09ahx7um5wgkhqatz9emk2mrvdaexgetj9ehx2aqc9kf8c)

**Screenshots/video recording**
![IMG_22E3D4C76728-1](https://github.com/damus-io/damus/assets/24692108/bd1af50f-7270-4038-9c7d-692582116b04)

** Versions **
Damus version: 1.9 (1)
Operating system version: iOS 17.3.1
Device: iPhone 13 Mini

**Steps To Reproduce**

1. Open Damus
2. Go to a note with a portrait video (like the one linked)
3. Click on the video to open the full screen carousel view.
4. See how the video looks like

**Additional context**
The current view works well with landscape videos. If increasing the height of the video player, we should be careful not to make the controls hidden behind the event text, and/or make it so big that we cannot tap outside of it to hide the event text.

**Comment by danieldaquino** [comment]
> yeah this is was referring to in our last standup

Yep! I am dumping my notes and stuff I saw on emails or conversations into Github issues to make sure we don't lose track of them

**Comment by jb55** [comment]
On Wed, Apr 17, 2024 at 01:02:39PM -0700, Daniel D’Aquino wrote:
>> yeah this is was referring to in our last standup
>
>Yep! I am dumping my notes and stuff I saw on emails or conversations into Github issues to make sure we don't lose track of them

I figured! just wanted to make sure.

**Comment by alltheseas** [comment]
Closing as part of a backlog triage — we're narrowing focus to the most actionable issues to reduce noise for contributors.

If this is still relevant, please reopen with updated context or reproduction steps. Thanks for reporting!
